environment.yml

```diff
@@ -0,0 +1,37 @@
+# To use:
+#   $ conda env create -f environment.yml  # `mamba` works too for this command
+#   $ conda activate scipy-dev
+name: scipy-dev
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - cython
+  - compilers
+  - numpy
+  - openblas
+  - libblas=*=*openblas  # helps avoid pulling in MKL
+  - pybind11
+  - pythran
```

Comment:
Should we add pythran dependencies, or leave it to the resolver to figure that out?

Reply:
Indirect dependencies are always for the resolver to deal with.